src/Controller/CardController.php

```diff
@@ -66,4 +67,15 @@ public function deleteCard(Card $card): Response
 
         return $this->json([], Response::HTTP_OK);
     }
+
+    /**
+     * Show Card
+     * @Route("/card/{id}",name="card_show",methods={"GET"})
+     */
+    public function showCard(ShowCard $showCard): Response
```

Comment:
You are create `Request/ShowCard` as QueryDTO its good. But you just return it in response, and that could be confusing while reading this for the first time. Of course this case is very simple so user will understand it quickly, but it still could be confusing. 

Try to make it more straightforward. For example, you could make field in this DTO
```
     private int $cardId
```
and handle it here, instead in ParamConverter:
```
$card = $this->cardRepository->getById($showCardQuery->getCardId());
``` 
(of course handle exception if not found and translate it to `NotFoundHttpException` in controller). 
With that your code will be more explicit. And param converter could handle raw validation eg. if user provide equal or grater cardId then '1' or if user provide int value

Reply:
Done, but i think we should discuss this.